wp-content/plugins/zippicks-favorites/includes/API/Client.php

```
@@ -0,0 +1,203 @@
+        return $this->post('/favorites', [
+            'user_id' => $user_id,
+            'business_id' => $business_id,
+            'source' => $context['source'] ?? 'wordpress',
+            'source_context' => $context
+        ]);
+    }
+    
+    /**
+     * Remove favorite
+     */
+    public function remove_favorite($user_id, $favorite_id) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$user_id'. (Unused Code Rules)

(UnusedFormalParameter)